tests/integration/test_reload_max_table_size_to_drop/test.py

```diff
@@ -22,7 +26,21 @@ def start_cluster():
         cluster.shutdown()
 
 
-def test_reload_max_table_size_to_drop(start_cluster):
+@pytest.fixture
```

Comment:
Actually I don't see the reason to write integration test for this feature, stateless test looks enough.

Reply:
Ignore this comment, we need to override defaults in config.xml, so integration test is required.